aspnet/DropDownList/Functionalities.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Functionalities in DropDownList | Syncfusion | ASP.NET WebForms
-description: Functionalities in the DropDownList control for Syncfusion ASP.NET WebForm
+title: Functionalities in ASP.NET Web Forms Dropdown List | Syncfusion
+description: Learn here all about Functionalities in Syncfusion ASP.NET Web Forms Dropdown List component of Syncfusion Essential JS 2 and more.
 platform: aspnet
```

Comment:
Learn here all about functionalities in Syncfusion ASP.NET Web Forms Dropdown List control, it's elements, and more.

Reply:
resolved